request_public_data.py

```diff
@@ -23,16 +25,19 @@
 if args.verbose:
 	logging.basicConfig(level=logging.DEBUG)
 
-r = requests.get("http://" + server_url + "/getData?from_date=" + from_date + "&to_date=" + to_date)
+from_date_UTC = arrow.get(from_date).replace(tzinfo='local').to('utc').format('YYYY-MM-DD HH:mm:ss')
```

Comment:
yea you are right... The conversion from `local time` to `UTC` is unnecessary. I thought you had to convert to `UTC` to get the correct timestamp for some reason, but that's wrong